neo/Prompt/Commands/Wallet.py

```diff
@@ -108,11 +107,10 @@ def execute(self, arguments):
 
```

Comment:
This case is still not covered. I think at least the anticipated case [here](https://coveralls.io/builds/20433094/source?filename=neo%2FPrompt%2FCommands%2FWallet.py#L101) should be tested.
I am not sure what the expected error is, but @ixje has mentioned to me in previous reviews that catchall try/except methods are undesirable. If this is still desirable logic, then I am okay with leaving it.
Also, if we don't want to cover this case then OK, I am just point it out.
https://coveralls.io/builds/20433094/source?filename=neo%2FPrompt%2FCommands%2FWallet.py#L102

Reply:
As a note, this is the copy of the code in the `development` branch.
I can force an exception for the sake of coverage.

Reply:
Right that is not what I'm going for explicitly. What it looks like is this specified case where `os.path.isfile(path)` is `True` isn't tested.

Reply:
Got it :+1: , I can make sure to cover these cases.